### PR TITLE
Get reviews

### PR DESCRIPTION
**Pull request by emmaambr**
Created reviewDB and rendered in spec movie page. Added simple styling.

### CODE REVIEW

**Comment by Nw84** [comment]
I am getting the following error when trying to look at the movie specifik pages
"error - MongoServerSelectionError: connection <monitor> to 16.170.60.90:27017 closed"

I think that you need to add access from all ip addresses to the database. 
network access and add 0.0.0.0/0

**Review by Nw84 — APPROVED** [review]
It works lika clock and it looks good. Good job! 👍 But there is a problem with overflow-y: scroll, but i did not see it when you shared your screen, so i think that it is a windows problem :)

File: components/reviews/ReviewItem.module.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+.name {
+    margin: auto;
+    font-size: 1.3rem;
+    font-weight: bold;
+}
+
+.comment {
+    overflow-wrap: break-word;
+    overflow-y: scroll;
```
Comment (Nw84) [inline components/reviews/ReviewItem.module.css:27]: I think that this causes the problem

Comment (emmaambr) [reply components/reviews/ReviewItem.module.css:27]: If we remove overflow-y, maybe we could replace it with a max number of characters in the review?

Comment (Nw84) [reply components/reviews/ReviewItem.module.css]: Yea that sounds like a good idea 👍  and it should work just adding a maxLength on the input

**Review by mhema42 — APPROVED** [review]
The reviews looks very fine and i can´t see any problem beside the coment Tobias made about overflow arrows in comment field.
I noticed that we need some "breadcrum" and to highlight "Filmer" in navbar on specific moviepage, but there is a scpecific ticket for "specific movie" so that problem must lie on that ticket :)
Very good job!

**Pull request merged**